corehq/apps/dashboard/views.py

```diff
@@ -49,22 +49,32 @@ def default_dashboard_url(request, domain):
     return reverse(DomainDashboardView.urlname, args=[domain])
 
 
-@login_and_domain_required
-def dashboard_tile(request, domain, slug):
+def _get_tile(request, slug):
     try:
         tile = [t for t in _get_default_tiles(request) if t.slug == slug][0]
     except IndexError:
         return json_response(
```

Comment:
I think this should raise a 404 exception - returning this will just confuse things in other places

Reply:
updated